Server/mods/deathmatch/logic/CScriptDebugging.h

```diff
@@ -73,6 +73,7 @@ class CScriptDebugging
                    unsigned char ucGreen = 255, unsigned char ucBlue = 255);
 
     void PrintLog(const char* szText);
+    bool CheckForSufficientDebugLevel(std::uint32_t playerDebugLevel, std::uint32_t messageDebugLevel) const noexcept;
```

Comment:
```std::uint8_t``` will be better in this case, we do not need such a large range and therefore take up 4 bytes of memory. uint8_t has a range from 0 to 255, which is sufficient here

Reply:
Done